api/src/main/resources/moduleApplicationContext.xml

```diff
@@ -11,71 +11,42 @@
            http://www.springframework.org/schema/util
            http://www.springframework.org/schema/util/spring-util-2.0.xsd">
 
-    <bean id="IdentifierSourceDAO" class="org.openmrs.module.idgen.service.db.HibernateIdentifierSourceDAO">
-        <property name="sessionFactory"><ref bean="dbSessionFactory"/></property>
+    <bean id="identifierSourceDAO" class="org.openmrs.module.idgen.service.db.HibernateIdentifierSourceDAO">
+        <property name="sessionFactory" ref="dbSessionFactory"/>
     </bean>
 
-	<bean id="baseIdentifierSourceService" class="org.openmrs.module.idgen.service.BaseIdentifierSourceService">
```

Comment:
Note, by removing this bean from the top-level, this introduces a backwards-incompatibility (which is an issue in the paperrecord module that PIH uses), but seems like a minor issue.